misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -133,6 +133,12 @@ class FakeRedis : Redis {
     }
   }
 
+  override fun hlen(key: String): Long {
+    synchronized(lock) {
```

Comment:
You should change this to just use `@Synchronized` on every function tbh, would avoid some indents

Reply:
Ha true. Honestly, just following the existing convention. Will remove the lock in a follow up.

Reply:
https://github.com/cashapp/misk/pull/2686